bin/amphtml-update.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+set -e
+
+# Go to the right location.
+if [[ '.' != $(dirname "$0") ]]; then
+	cd bin
+fi
+
+BIN_PATH="$(pwd)"
+PROJECT_PATH=$(dirname $PWD)
+VENDOR_PATH=$PROJECT_PATH/vendor
+
+if ! apt-get > /dev/null; then
+	echo -e "The AMP HTML uses the apt-get, make sure to run this script in a Linux environment"
+	exit
+fi
+
+# Install dependencies.
+sudo apt-get install git
+sudo apt-get install python
```

Comment:
Maybe we should make `git` and `python` just dependencies that we assume are installed, and exit if the command doesn't exist?

```bash
if ! command -v git >/dev/null 2>&1; then
    echo "Git is required"
    exit 1
fi
if ! command -v python >/dev/null 2>&1; then
    echo "Python is required"
    exit 1
fi
```

Reply:
I think it is fine to automatically install `git` or `python` if not installed, the check is pretty quick if it is already installed.